cmd/fetcher/app/server.go

```diff
@@ -115,6 +115,8 @@ func Run(logger *zap.Logger) {
 	mux.HandleFunc("/specialize", f.SpecializeHandler)
 	mux.HandleFunc("/upload", f.UploadHandler)
 	mux.HandleFunc("/version", f.VersionHandler)
+	mux.HandleFunc("/websocket", f.WebsocketHandler)
```

Comment:
I think `websocket` is a pretty generic and misleading name for this endpoint. I would use something that represents the fact that we are sending an event to indicate that the environment has started some long-running activity - and don't reap me. Something like `/start_long_connection` or something of that sort please.

Reply:
The event name itself needs to be in camel case. So I'm thinking the end point name should be just `longlivedConnection`  and the event name can be `InitiateLongLivedConnection`. WDYT?

Reply:
`LongLivedConnection` seems a bit too specific. How about `/eventStart` as the name of endpoint - which is to indicate that you are indicating the start of an event. The actual event could be `wsConnStartedEvent`?